src/overlays/actors/ovl_Obj_Switch/z_obj_switch.h

```diff
@@ -17,26 +17,16 @@ typedef enum {
 } ObjSwitchType;
 
 typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_FLOOR_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_FLOOR_1,
-    /* 2 */ OBJSWITCH_SUBTYPE_FLOOR_2,
-    /* 3 */ OBJSWITCH_SUBTYPE_FLOOR_3
-} ObjSwitchSubTypeFloor;
-
-typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_EYE_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_EYE_1
-} ObjSwitchSubTypeEye;
-
-typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_CRYSTAL_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_CRYSTAL_1,
-    /* 4 */ OBJSWITCH_SUBTYPE_CRYSTAL_4 = 4
-} ObjSwitchSubTypeCrystal;
+    /* 0 */ OBJSWITCH_SUBTYPE_DEFAULT,      // Normal Yellow Switches
```

Comment:
```suggestion
    /* 0 */ OBJSWITCH_SUBTYPE_ONCE, // Switches that can only be turned on
```
not sure about "once" tbh, but I really don't like "default"

Reply:
I wasn't sure about default too, I can't find the best word for this, I'll take "once" for now and if someone got a better name then I'll update it